walk_vel: 0.1
run_vel: 0.3

walk_vel_y: 0.2
run_vel_y: 0.2

yaw_rate: 0.1
yaw_rate_run: 0.3

